docs/how-to/command-line-quick-start.md

```diff
@@ -16,9 +16,9 @@ Don’t want to use the command line right now? Give the desktop-app implementat
 
 Installing IPFS is simple, but varies between operating system:
 
-| [Windows](#windows) | [macOS](#macos) | [Linux](#linux) |
-| --- | --- | --- |
-| [![Windows icon](./images/command-line-quick-start/windows-icon.png)](#windows) | [![macOS icon](./images/command-line-quick-start/apple-icon.png)](#macos) | [![Linux icon](./images/command-line-quick-start/linux-icon.png)](#linux) |
+| [Windows](#windows)                                                                                 | [macOS](#macos)                                                                               | [Linux](#linux)                                                                               |
+| --------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
```

Comment:
Do you have a markdown formatter that's introducing this spacing? The previous spacing was easier to read in raw format.

Reply:
the previous formatting was made by me manually something is adding this 

```
"lint-staged": {
    "*.{js,json,css,less,scss,html,md,vue}": [
      "prettier --write"
    ],
    "*.styl": [
      "stylus-supremacy format --replace"
    ]
  },
```